src/components/help/HelpContainer.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { ReactComponent as ExitSVG } from 'media/icons/times-light.svg';
+import { ReactComponent as QuestionCircleSVG } from 'media/icons/question-circle-solid.svg';
+import HelpPanel from './HelpPanel';
+
+const styles = (theme) => ({
+  panelContainer: {
+    color: '#333F52',
```

Comment:
I think we want colors like this to live in theme.js (if it's not there already)